src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -201,6 +202,21 @@ suggestRemoveRedundantImport ParsedModule{pm_parsed_source = L _  HsModule{hsmod
         = [("Remove import", [TextEdit (extendToWholeLineIfPossible contents _range) ""])]
     | otherwise = []
 
+removeAllRedundantImports :: Maybe ParsedModule -> Maybe T.Text -> [Diagnostic] -> Uri -> [CAResult]
+removeAllRedundantImports m contents diagnostics uri
+  | Just pm <- m,
+    r <- join $ map (suggestRemoveRedundantImport pm contents) diagnostics,
```

Comment:
Is there any way to avoid calling `suggestRemoveRedundantImport` twice of every diagnostic?